internals/secrethub/org.go

```diff
@@ -22,6 +22,7 @@ func (cmd *OrgCommand) Register(r Registerer) {
 	NewOrgInitCommand(cmd.io, cmd.newClient).Register(clause)
 	NewOrgInspectCommand(cmd.io, cmd.newClient).Register(clause)
 	NewOrgInviteCommand(cmd.io, cmd.newClient).Register(clause)
+	NewOrgPurchaseCommand(cmd.io).Register(clause)
```

Comment:
Should it be on the `org` level? Could also just be `secrethub purchase`, just like `secrethub signup`.

Reply:
A subscription is purchased by and for an organization. Users that do not use organizations don't have to purchase, as personal accounts are free. Therefore, I think the purchase command should be a subcommand of `org`. This also keeps the `--help` output relevant. `purchase` is not relevant for users that do not use organizations.

Reply:
Okay, though we could it the future make more non-org features premium as well. But since you won't script this command anyway, it's fine to break it if we ever decide to do that.

Reply:
I don't think that's the intention.

Even if it were, we can add a `purchase` command at root level later and add a (hidden) alias from this command. No need to break :smile: